packages/core/src/accessanalyzer/vue/constants.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export abstract class IamPolicyChecksConstants {
+    static readonly ValidatePolicySuccessWithFindings =
+        'Please view the problems panel to review the issues with your policy document. Policy checks should be run until issues are no longer found in your policy document.'
+    static readonly ValidatePolicySuccessNoFindings = 'Policy checks did not discover any problems with your policy.'
+    static readonly CustomCheckSuccessWithFindings =
+        'Result: FAIL. Please view the problems panel to review the issues with your policy document. Policy checks should be run until issues are no longer found in your policy document.'
+    static readonly CustomCheckSuccessNoFindings =
+        'Result: PASS. Policy checks did not discover any problems with your policy.'
+    static readonly CustomCheckFilePathSetting = 'aws.accessAnalyzer.policyChecks.customChecksFilePath'
+    static readonly CfnParameterFilePathSetting = 'aws.accessAnalyzer.policyChecks.cloudFormationParameterFilePath'
+    static readonly MissingReferenceDocError = 'Reference policy document is missing.'
+    static readonly IncorrectFileExtension =
+        'The file extension does not match the selected document type. Please select the correct document type, or use the proper file extension.'
+    static readonly InvalidAwsCredentials = 'AWS Role Credentials are invalid. Update AWS credentials and try again.'
+}
+
+export enum PolicyChecksErrorCode {
+    FileReadError = 'FileReadError',
+}
+
+export enum PolicyChecksDocumentType {
+    TERRAFORM_PLAN = 'Terraform Plan',
+    CLOUDFORMATION = 'CloudFormation',
+    JSON_POLICY_LANGUAGE = 'JSON Policy Language',
+}
+
+export enum ValidatePolicyFindingType {
+    ERROR = 'ERROR',
+    SECURITY_WARNING = 'SECURITY_WARNING',
+    SUGGESTION = 'SUGGESTION',
+    WARNING = 'WARNING',
+}
```

Comment:
enums really aren't needed in modern typescript. instead it's preferred to use this pattern: https://github.com/aws/aws-toolkit-vscode/blob/1ba1e0427f4e797c1124bc12c5bd6addfec5138f/packages/core/src/login/webview/vue/reauthenticate.vue#L85-L89